tensorflow_recommenders_addons/utils/resource_loader.py

```diff
@@ -103,6 +142,26 @@ def abi_is_compatible():
   if "dev" in tf.__version__:
     return False
 
-  min_version = LooseVersion(MIN_TF_VERSION_FOR_ABI_COMPATIBILITY)
-  max_version = LooseVersion(MAX_TF_VERSION_FOR_ABI_COMPATIBILITY)
-  return min_version <= LooseVersion(tf.__version__) <= max_version
+  required_tf_version = get_required_tf_version()
+  return tf.__version__ == required_tf_version
+
+
+def decorate_op_name(op_name):
+  """
+  In order to keep compatibility of existing models,
+  we cannot change the OP naming rule directly by replacing "TFRA>" to "Tfra",
+  So we had to add prefix to OP name according to the TF verison.
+
+  Args:
+    op_name: original OP name
+  Returns:
+    OP name with prefix
+  """
+  major_tf_version = int(tf.__version__.split(".")[0])
+  _prefix = "TFRA>" if major_tf_version >= 2 else "Tfra"
+  return _prefix + op_name
+
+
+def get_tf_version_triple():
```

Comment:
I think it will more precise with [semantic version](https://semver.org/spec/v2.0.0.html)